bundles/org.openhab.core.semantics/src/main/java/org/openhab/core/semantics/Property.java

```diff
@@ -20,6 +20,6 @@
  * @author Kai Kreuzer - Initial contribution
  */
 @NonNullByDefault
-@TagInfo(id = "MeasurementProperty")
```

Comment:
@kaikreuzer was this an unintentional typo?

Reply:
I have no clue. It might be that `MeasurementProperty` was the original name and we missed to update that line when we simplified the name to `Property`.